.github/workflows/pull-request.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches: [ main ]
 
+    # Run this workflow manually from the Actions tab
+    workflow_dispatch:
```

Comment:
I don't understand this, isn't it missing a parameter at least?

Reply:
This is the same as we have in MIEngine for allowing manual triggers for this Action.

https://github.com/microsoft/MIEngine/blob/34db09e636d79395b6d372faf52a66727a2220fe/.github/workflows/Build-And-Test.yml#L13


More Information: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch